crypto/rand/rand_lib.c

```diff
@@ -566,8 +566,7 @@ size_t rand_pool_bytes_needed(RAND_POOL *pool, unsigned int entropy_factor)
         return 0;
     }
 
-    if (pool->len < pool->min_len &&
-        bytes_needed < pool->min_len - pool->len)
+    if (pool->len < pool->min_len && bytes_needed < pool->min_len - pool->len)
```

Comment:
I don't see any necessity to make this unrelated whitespace-only change. It only distracts from the real change. With GitHub fine-diffs it's no problem to see that it's whitespace-only, but otherwise one has to look twice.